src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -72,11 +66,11 @@ private static List<TableOfContentsInputItem> LoadTocModel(string content, strin
                 {
                     throw new NotSupportedException($"{filePath} is not a valid TOC file, detail: {ex.Message}.", ex);
```

Comment:
>throw new NotSupportedException($"{filePath} is not a valid TOC file, detail: {ex.Message}.", ex); [](start = 20, length = 98)

also move this to error?

Reply:
?

---
In reply to: [198069924](https://github.com/dotnet/docfx/pull/2943#discussion_r198069924) [](ancestors = 198069924)